plugins/inputs/http_listener_v2/http_listener_v2_test.go

```diff
@@ -592,7 +593,11 @@ func TestWriteHTTPTransformHeaderValuesToTagsBulkWrite(t *testing.T) {
 }
 
 func TestWriteHTTPQueryParams(t *testing.T) {
-	parser, _ := parsers.NewFormUrlencodedParser("query_measurement", nil, []string{"tagKey"})
+	parser := form_urlencoded.Parser{}
+	require.NoError(t, parser.Init())
+	parser.MetricName = "query_measurement"
+	parser.TagKeys = []string{"tagKey"}
```

Comment:
Shouldn't this be
```suggestion
	parser := form_urlencoded.Parser{
		MetricName: "query_measurement"
		TagKeys: []string{"tagKey"}
	}
	require.NoError(t, parser.Init())
```

Same below...